client/src/client_sync/v23/wallet.rs

```diff
@@ -15,6 +15,8 @@ macro_rules! impl_client_v23__create_wallet {
     () => {
         impl Client {
             /// Calls `createwallet` with `wallet` as the only argument.
+            /// In v23 and later this creates a descriptor wallet. Use `create_legacy_wallet` to create
+            /// a legacy wallet.
```

Comment:
```suggestion
            /// Calls `createwallet` with `wallet` as the only argument.
            ///
            /// In v23 and later this creates a descriptor wallet. Use `create_legacy_wallet` to create
            /// a legacy wallet.
```

Bit nit-picky and I wouldn't bother rebasing just for this.

Reply:
Done